primary-maternal-care/1B.md

```diff
@@ -1,210 +1,878 @@
 ---
 book: Primary Maternal Care
-title: 1B. Examination of the abdomen in pregnancy
+title: 1. Antenatal care
```

Comment:
@nkoeries This looks like an old Skills chapter (1B) has been replaced with one of the updated chapters taken from Maternal Care (chapter 1). Was this a mistake or am I missing something?